docs/quickstart.md

```diff
@@ -321,3 +383,43 @@ spark.sql("SELECT * FROM quickstart_table").show(false)
 
 org.apache.iceberg.exceptions.ForbiddenException: Forbidden: Principal 'quickstart_user' with activated PrincipalRoles '[]' and activated ids '[6, 7]' is not authorized for op LOAD_TABLE_WITH_READ_DELEGATION
 ```
+
+### Connecting with Trino
+
+To use a Polaris-managed catalog in [Trino](https://trino.io/), you can
+configure Trino to use the Iceberg REST API.
+
+Generate and export a `PRINCIPAL_TOKEN` per the [README](/README.md#creating-a-catalog-manually),
+but create a Polaris catalog with the following specifications:
+
+```
+curl -i -X POST -H "Authorization: Bearer $PRINCIPAL_TOKEN" -H 'Accept: application/json' -H 'Content-Type: application/json' \
+http://localhost:8181/api/management/v1/catalogs \
+-d '{"name": "polaris", "id": 100, "type": "INTERNAL", "readOnly": false, "storageConfigInfo": {"storageType": "FILE"}, "properties": {"default-base-location": "file:///tmp/path_to_catalog"}}'
+```
+
+You can then show catalogs with:
+
+```
+curl -i -X GET -H "Authorization: Bearer $PRINCIPAL_TOKEN" -H 'Accept: application/json' -H 'Content-Type: application/json' \
+http://localhost:8181/api/management/v1/catalogs
+```
+
+You'll need to have Trino installed, so download the [latest version of Trino](https://trino.io/download),
+and you can follow [the Trino docs](https://trino.io/docs/current/installation.html)
+to install it.
+
+Once Trino is installed, create a catalog properties file, `polaris.properties`,
+in the `etc/catalog/` directory of your Trino installation. This is the file
+where you can configure Trino's Iceberg connector. Edit it to:
+
+```
+connector.name=iceberg
+iceberg.catalog.type=rest
+iceberg.rest-catalog.security=OAUTH2
+iceberg.rest-catalog.oauth2.token={the value of your PRINCIPAL_TOKEN}
+iceberg.rest-catalog.warehouse={the catalog name created in the above POST request, e.g. 'polaris'}
+```
+
+Start (or restart) Trino, and `SHOW CATALOGS` should show the Polaris catalog.
+You can then run `USE polaris` to access, query, or write to Polaris.
```

Comment:
When I tried this, I had trouble creating a table. Let's make sure the quickstart contains a working end-to-end example